src/Test/PdbUtilities/Shared/DateTimeUtilities.cs

```diff
@@ -6,7 +6,7 @@ namespace Roslyn.Utilities
 {
     internal static class DateTimeUtilities
     {
-        internal const string DateTimeDateDataFieldName = "dateData";
+        internal const string DateTimeInternalTicksPropertyName = "InternalTicks";
```

Comment:
Can we use the `Ticks` property instead?

Reply:
Also, it looks like this `const` is only used in the EE. Consider moving it there.